src/server.h

```diff
@@ -300,6 +301,22 @@ class Server : public con::PeerHandler, public MapEventReceiver,
 		setAsyncFatalError(std::string("Lua: ") + e.what());
 	}
 
+	// Not thread-safe.
+	inline void addShutdownError(const ModError &e) {
```

Comment:
Would you please be so nice to move this function to `server.cpp`? Headers should be kept lightweight.

Reply:
Sure, done :)